tutorials/install.md

```diff
@@ -60,6 +89,23 @@ Add OSRF packages:
     wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
     sudo apt-add-repository -s "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -c -s) main"
 
+Only on Bionic, update the GCC compiler version:
+
+    sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8
+
+#### Windows 10
+
+1. Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+   Miniconda suffices.
+
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/).
+   The Community version is free for students, open-source contributors, and individuals.
```

Comment:
`individuals` sounds a bit awkward here, perhaps personal development?

Reply:
And also perhaps open-source contributors -> open-source developers?  The first implies to me that a contribution has to have been made to get it for free.

Reply:
I used the wording verbatim from the Microsoft webpage, which lists Community as "Powerful IDE, free for students, open-source contributors, and individuals." I thought that was the most political correct to just say what they say.

Reply:
Sounds good to me!